common/xfdesktop-thumbnailer.c, src/xfdesktop-file-utils.c and 'src/xfdesktop-file-utils.h
always use #include <dbus/dbus-glib.h> but configure.ac doesn't check for it if
configured with flags --disable-file-icons --disable-thunarx, causing $(DBUS_CFLAGS)
to be unset and build to fail on a missing -I/usr/include/dbus-1.0

http://bugzilla.xfce.org/show_bug.cgi?id=10745

--- configure.ac
+++ configure.ac
@@ -87,6 +87,7 @@
 XDT_CHECK_PACKAGE([GOBJECT], [gobject-2.0], [glib_minimum_version])
 XDT_CHECK_PACKAGE([GTHREAD], [gthread-2.0], [glib_minimum_version])
 XDT_CHECK_PACKAGE([GTK], [gtk+-2.0], [gtk_minimum_version])
+XDT_CHECK_PACKAGE([DBUS], [dbus-glib-1], [dbus_minimum_version])
 XDT_CHECK_PACKAGE([LIBXFCE4UTIL], [libxfce4util-1.0],
                   [libxfce4util_minimum_version])
 XDT_CHECK_PACKAGE([LIBXFCE4UI], [libxfce4ui-1],
@@ -124,14 +125,6 @@
 fi
 
 if test "x$enable_desktop_icons" = "xyes" -a "x$enable_file_icons" = "xyes"; then
-    dnl here i shall abuse the XDT_CHECK_PACKAGE macro
-    XDT_CHECK_PACKAGE([DBUS], [dbus-glib-1], [dbus_minimum_version], [], [
-echo "*** Optional package dbus-glib-1 was either not found on your system"
-echo "*** or is too old.  Please install or upgrade to at least version"
-echo "*** dbus_minimum_version, or adjust the PKG_CONFIG_PATH environment variable"
-echo "*** if you installed the new version of the package in a nonstandard"
-echo "*** prefix.  File/launcher icons will be disabled."
-])
     enable_file_icons="no"
     if test "x$DBUS_VERSION" != "x"; then
         enable_file_icons="yes"
